Makefile

```diff
@@ -153,7 +153,7 @@ integration-test-run-sequential:
 	@go test -timeout 20m ./tests/endpoints/...
 	@go test -timeout 20m ./tests/env/... --logtostderr
 	@go test -timeout 20m ./tests/utils/... --logtostderr
-	@go test -timeout 50m -parallel 1 ./tests/integration_test --logtostderr
+	@go test -v -timeout 80m -parallel 1 ./tests/integration_test --logtostderr
```

Comment:
These are intentional. 
- We were close to hitting the timeouts, increase them some more.
- `-v` to display all logs even when there is no test failure. I find this helpful, and having more logs does not hurt.